src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ApplicationIntent.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+    /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/ApplicationIntent.xml' path='docs/members[@name="ApplicationIntent"]/ApplicationIntent/*'/>
+#if NETFRAMEWORK
```

Comment:
`NETFRAMEWORK` isn't recognized for .NET Framework implicitly.
You'll need to add `DefineConstants` in csproj and register this constant (similar to [here](https://github.com/dotnet/SqlClient/pull/597/files#diff-eb4be71042e9e5d81f7b29c13ab6d27cR23)).
Would recommend registering as `netfx` (uniform with folder arch and existing constants).

Reply:
@Wraith2 we need the constants applied in solution, which doesn't seem to happen when working in Visual Studio.

![image](https://user-images.githubusercontent.com/13396919/85423614-c6197600-b52b-11ea-8bba-4282325359bd.png)

Including it in csproj will guarantee that.

Reply:
If you use /bl to get the binlog you can look at the full set of defines that are produced and you'll find NETFRAMEWORK is in there only if you're building for a framework target. There's a similar one for netcore but I didn't need that. So it doesn't need to be defined because it's already an integral part of the build system multi framework targeting.

Reply:
I believe that's true, but my concern is for developer experience. If we continue to use this for future developments, it would never load source code that applies to .NET Framework in IDE when testing/debugging applications. So maybe IDE is to blame, but that constant isn't available by default in Visual Studio.

Reply:
Ok, define added.